xml_converter/refresh.sh

```diff
@@ -0,0 +1,13 @@
+cd ..
```

Comment:
I thought I had this on ignore. I'll fix this

Reply:
Other people have disagreed with me in the past, even trying to "teach" me how to use `git add *` to add files faster, but I have a compulsion to do a `git diff <file>` `git add <file>` for every file I am staging for commit so that I know actually what changes and files I am putting into the commit. I personally do this because it means that my reviewer is not the first human to look at my changes, and instead I am doing the first pass to catch any easy-to-find issues in my own work.